packages/forms/src/ErrorMessage.tsx

```diff
@@ -13,10 +26,28 @@ export const ErrorMessage = <TFieldValues extends FieldValues = FieldValues>({
     formState: { errors },
   } = useFormContext<TFieldValues>();
 
+  const {
+    intl: { formatMessage },
+  } = useI18n();
+
+  const error = errors[name];
+
+  if (!error) return null;
+
+  const { message } = error;
+
+  const i18nMessage = isMessageDescriptor(message)
+    ? formatMessage(message, message?.values)
+    : error;
+
+  const singleErrorMessage: any = {};
+  singleErrorMessage[name] = i18nMessage;
```

Comment:
We don't need `HookFormErrorMessage` if we handle `singleErrorMessage` by ourselves.